cdap-common/bin/functions.sh

```diff
@@ -413,7 +413,11 @@ cdap_set_hbase() {
     1.2-cdh*) __compat="${CDAP_HOME}"/hbase-compat-1.2-cdh5.7.0/lib/* ;; # 5.7 and 5.8 are compatible
     1.2*) __compat="${CDAP_HOME}"/hbase-compat-1.1/lib/* ;; # 1.1 and 1.2 are compatible
     "") die "Unable to determine HBase version! Aborting." ;;
-    *) die "Unknown/Unsupported HBase version found: ${HBASE_VERSION}" ;;
+    *) if [[ $(cdap_get_conf "hbase.latest.version.for.unknown.version" "${CDAP_CONF}"/cdap-site.xml false) == 'true' ]]; then
+       __compat="${CDAP_HOME}"/hbase-compat-1.2-cdh5.7.0/lib/* # if allowed, return the latest HBase version for unknown version
```

Comment:
@wolf31o2 should I break the line before `if` or should just add two spaces before `__compat=...`?